### DISCUSSION TITLE
Failing to upload any files (any help appreciated)

### DISCUSSION THREAD

**Discussion by jbedgood** [mastodon/mastodon#22237] [General]
I've just installed mastodon on debian 11. The logins work and signing up works, but none of the file uploads is working and I'm lost as to where to look as there is nothing showing up in the log files for nginx.

This is a straight install not docker. The main mastodon directory and all directories under it are owned by the mastodon user/group. Yet the error i get when I try to upload an avatar or even a file for the main site logo is "we're sorry but something went wrong on our end." and thats it no code or nothing. So obviously no help in tracking down where to look

**Comment by wiegelmann** [comment]
Check the logs for web and background jobs

journalctl -f -u mastodon-web.service
journalctl -f -u mastodon-sidekiq.service
